notebooks/Imputation_best_practices/Imputation_best_practices.ipynb

```diff
@@ -0,0 +1,4557 @@
+{
```

Comment:
<p>Can we pivot this df so that the columns are <code>1%_saleprice</code>,<code>5%_saleprice</code>,and <code>10%_saleprice</code> , and the rows are the <code>diff.list Mean(&lt;method&gt;)</code>? This way we wouldn't see these nan cells in the df (iiuc). wdyt?       </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflows/pull/113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='142'/>

Reply:
<p>Will Use multi-table comparison for a better presentation!</p>